src/php/Evaluator/MethodNodeEvaluator/Contract/RouterTest.php

```diff
@@ -0,0 +1,294 @@
+<?php
+
+namespace Viamo\Floip\Evaluator\MethodNodeEvaluator\Contract;
+
+use Viamo\Floip\Evaluator\MethodNodeEvaluator\Contract\EvaluatesMethods;
+use Viamo\Floip\Evaluator\MethodNodeEvaluator\Contract\TestResult;
+
+interface RouterTest extends EvaluatesMethods
```

Comment:
Nice work @seifertk !  One small thing to consider on naming: Flow Interop doesn't have the concept of a "router"; this is only in the RapidPro world.  However, we do need to support all of the test expressions that RapidPro might use in routers... They would come into Exit test expressions in the Flow world.  For that reason, the "RouterTest" class naming might not be ideal here, but it's not a big problem.

Reply:
It's very easy to rename at this point: any suggestions?

Reply:
Nope; I trust your creativity :)